test/integration/command/runner_test.py

```diff
@@ -34,7 +34,135 @@ def dockerfile_path(name=None):
 
 
 @pytest.mark.skipif(docker_skip('ubuntu:22.04'), reason="Only docker running")
-# @pytest.mark.xfail(reason="conan inside docker optional test")
+def test_create_docker_runner_cache_shared():
```

Comment:
these tests are repeating the same code, maybe we can define those in a common place or parameterize the tests?